Source/WebCore/platform/graphics/cg/ImageBufferIOSurfaceBackend.cpp

```diff
@@ -43,6 +43,40 @@ namespace WebCore {
 
 WTF_MAKE_ISO_ALLOCATED_IMPL(ImageBufferIOSurfaceBackend);
 
+static RetainPtr<CGImageRef> createIOSurfaceBitmapImageReference(IOSurface& surface)
```

Comment:
I think including `BitmapImage` in the name is a little bit confusing. Also the name does not reveal the nature of the returned `CGImage`. I would suggest adding this function to `IOSurface` or adding it as a new mode to `IOSurface::createImage(BackingStoreCopy copyBehavior)`. If `copyBehavior == DontCopyBackingStore`, it call this new function.